src/svcutilcore/files/dotnet-svcutil.xmlserializer.targets

```diff
@@ -1,6 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
- <PropertyGroup>
-   <MyProperty>$(IntermediateOutputPath)</MyProperty>
+  <PropertyGroup>
+    <_IsNet5Compatable>$([MSBuild]::IsTargetFrameworkCompatible('net5.0-windows', 'net5.0'))</_IsNet5Compatable>
```

Comment:
>IsTargetFrameworkCompatible [](http://example.com/codeflow?start=36&length=27)

This looks like this would always return true as net5.0 is compatible with net5.0-windows. Would I be right to presume it can't answer this question if sdk 5.0 hasn't been installed so will return false in that case?